infra/ansible/config/env.yaml

```diff
@@ -42,7 +42,9 @@ build_env:
 
   cuda:
     TF_CUDA_COMPUTE_CAPABILITIES: "{{ cuda_compute_capabilities }}"
+    TORCH_CUDA_ARCH_LIST: "{{ torch_cuda_arch_list | default('') }}"
     XLA_CUDA: 1
+    USE_CUDA: "{{ build_pytorch_with_cuda | default(0) }}"
```

Comment:
Discussed offline. Please rebase onto master and undo the build/CI infra changes, and then I'll work with you on a follow up PR to support the OpenXLA CUDA plugin.

Reply:
Reverted the build/CI infra changes. Thanks for helping me out with this